benchmarks/cugraph/standalone/bulk_sampling/datasets/ogbn_papers100M.py

```diff
@@ -24,6 +24,10 @@
 import os
 import json
 
+from cugraph.utilities.utils import import_optional
+
+wgth = import_optional("pylibwholegraph.torch")
```

Comment:
Just double checking beacuse i dont know enough about `pylibwholegraph.torch`

This does not create any Pytorch context right on import otherwise it may cause problems with `//` trainers.